src/dune_config/config.mli

```diff
@@ -67,3 +69,5 @@ val threaded_console_frames_per_second : [ `Default | `Custom of int ] t
     Note that environment variables take precedence over the values passed here
     for easy overriding. *)
 val init : (Loc.t * string) String.Map.t -> unit
+
+val party_mode : Toggle.t t
```

Comment:
This is just a dummy config, do you still need it after testing?

Reply:
No, this is just for testing, it can safely be removed. I haven't considered it to be of particular importance but it can be reverted without any problem.